cmake/zqConfig.cmake.in

```diff
@@ -2,4 +2,10 @@
 
 include("${CMAKE_CURRENT_LIST_DIR}/zqTargets.cmake")
 
+include(CMakeFindDependencyMacro)
+find_dependency(astr)
+find_dependency(Protobuf)
```

Comment:
do you have any suggestion on how that could work together with this PR
https://github.com/a4z/zq/pull/5
which makes Protobuf optional?

Reply:
there are also other scenarios, like, when passing NO_CMAKE_FIND_ROOT_PATH to find package in some build configurations ....

Reply:
We could add 
`set(Protobuf_DEPENDENCY "find_dependency(Protobuf)")` inside if statement https://github.com/a4z/zq/blob/d8de964359887c493f07bc746a76b39e79e00335/CMakeLists.txt#L76

and replace `find_dependency(Protobuf)` with `@Protobuf_DEPENDENCY@` in https://github.com/a4z/zq/blob/d8de964359887c493f07bc746a76b39e79e00335/cmake/zqConfig.cmake.in. 
That way, cmake will only look for Protobuf if the variable is set.

Reply:
sounds like a good idea!

is it ok if I redirect the merge request to the feature/optional_proto branch so it goes together with https://github.com/a4z/zq/pull/5 as one change?